metaserver/src/main/scala/scala/meta/languageserver/providers/ReferencesProvider.scala

```diff
@@ -16,10 +16,7 @@ object ReferencesProvider extends LazyLogging {
       context: l.ReferenceContext
   ): ReferencesResult = {
     val locations = for {
-      symbol <- symbolIndex
-        .findSymbol(uri, position.line, position.character)
-        .toList
-      data <- symbolIndex.referencesData(symbol)
+      data <- symbolIndex.findReferences(uri, position.line, position.character)
```

Comment:
I’m not sure if this is needed for performance but I don’t like passing around line, character everywhere.  It feels like passing x, y instead of Point.  An indicator of a missing data structure.

Reply:
I totally agree, been meaning to clean it up at some point but until then I prefer to keep things consistent to make the migration easier once we get around to improve this part.